### PR TITLE
fix: object creation broke React memoization in DetailsList

### PR DESCRIPTION
**Pull request by spmonahan**
## Previous Behavior

Props objects that never changed (i.e., keys and values are always the same) were created in React's `render()` path, preventing memoization from functioning properly.

## New Behavior

Objects are created at the module root.

Additionally, linting rules were applied.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/d1a7eb2d7768ccb46293ab261055fc61634ee70b'>d1a7eb2d7768ccb46293ab261055fc61634ee70b</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/ContextualMenu-mount_base.html">16632</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/ContextualMenu-mount.html">8018</a></td>
            <td>1000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/ContextualMenu-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/GroupedList-mount_base.html">2191</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/GroupedList-mount.html">2516</a></td>
            <td>2</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/GroupedList-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/BaseButton-mount_base.html">1410</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/BaseButton-mount.html">1388</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Breadcrumb-mount_base.html">3486</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Breadcrumb-mount.html">3544</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Checkbox-mount_base.html">3109</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Checkbox-mount.html">3130</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/CheckboxBase-mount_base.html">2817</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/CheckboxBase-mount.html">2782</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/ChoiceGroup-mount_base.html">5253</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/ChoiceGroup-mount.html">5232</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/ComboBox-mount_base.html">1484</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/ComboBox-mount.html">1494</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/CommandBar-mount_base.html">11339</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/CommandBar-mount.html">11779</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/ContextualMenu-mount_base.html">16632</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/ContextualMenu-mount.html">8018</a></td>
            <td>1000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/ContextualMenu-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/DefaultButton-mount_base.html">1610</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/DefaultButton-mount.html">1665</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/DetailsRow-mount_base.html">4127</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/DetailsRow-mount.html">4060</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/DetailsRowFast-mount_base.html">4004</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/DetailsRowFast-mount.html">4079</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/DetailsRowNoStyles-mount_base.html">3892</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/DetailsRowNoStyles-mount.html">4076</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Dialog-mount_base.html">3632</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Dialog-mount.html">3672</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/DocumentCardTitle-mount_base.html">696</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/DocumentCardTitle-mount.html">680</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Dropdown-mount_base.html">3800</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Dropdown-mount.html">3812</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/FocusTrapZone-mount_base.html">2392</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/FocusTrapZone-mount.html">2391</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/FocusZone-mount_base.html">2362</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/FocusZone-mount.html">2310</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/GroupedList-mount_base.html">2191</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/GroupedList-mount.html">2516</a></td>
            <td>2</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/GroupedList-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/GroupedList-virtual-rerender_base.html">1358</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/GroupedList-virtual-rerender.html">1354</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">1921</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/GroupedList-virtual-rerender-with-unmount.html">1935</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/GroupedListV2-mount_base.html">689</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/GroupedListV2-mount.html">686</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/GroupedListV2-virtual-rerender_base.html">653</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/GroupedListV2-virtual-rerender.html">641</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">664</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">686</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/IconButton-mount_base.html">2247</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/IconButton-mount.html">2246</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Label-mount_base.html">888</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Label-mount.html">872</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Layer-mount_base.html">5093</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Layer-mount.html">5155</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Link-mount_base.html">973</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Link-mount.html">970</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/MenuButton-mount_base.html">1970</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/MenuButton-mount.html">1992</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/MessageBar-mount_base.html">2753</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/MessageBar-mount.html">2750</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Nav-mount_base.html">3857</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Nav-mount.html">3858</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/OverflowSet-mount_base.html">1589</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/OverflowSet-mount.html">1593</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Panel-mount_base.html">2979</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Panel-mount.html">3039</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Persona-mount_base.html">1548</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Persona-mount.html">1533</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Pivot-mount_base.html">1967</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Pivot-mount.html">1974</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/PrimaryButton-mount_base.html">1783</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/PrimaryButton-mount.html">1826</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Rating-mount_base.html">8143</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Rating-mount.html">8209</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/SearchBox-mount_base.html">1770</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/SearchBox-mount.html">1791</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Shimmer-mount_base.html">3323</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Shimmer-mount.html">3279</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Slider-mount_base.html">2449</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Slider-mount.html">2440</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/SpinButton-mount_base.html">5446</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/SpinButton-mount.html">5493</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Spinner-mount_base.html">927</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Spinner-mount.html">936</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/SplitButton-mount_base.html">3605</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/SplitButton-mount.html">3723</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Stack-mount_base.html">965</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Stack-mount.html">980</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/StackWithIntrinsicChildren-mount_base.html">2702</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/StackWithIntrinsicChildren-mount.html">2697</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/StackWithTextChildren-mount_base.html">5588</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/StackWithTextChildren-mount.html">5580</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/SwatchColorPicker-mount_base.html">12296</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/SwatchColorPicker-mount.html">12348</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/TagPicker-mount_base.html">3101</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/TagPicker-mount.html">3074</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/TeachingBubble-mount_base.html">100150</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/TeachingBubble-mount.html">101073</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Text-mount_base.html">943</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Text-mount.html">942</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/TextField-mount_base.html">1886</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/TextField-mount.html">1900</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/ThemeProvider-mount_base.html">1815</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/ThemeProvider-mount.html">1834</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/ThemeProvider-virtual-rerender_base.html">1302</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/ThemeProvider-virtual-rerender.html">1277</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">2568</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2584</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Toggle-mount_base.html">1341</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/Toggle-mount.html">1287</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/buttonNative-mount_base.html">638</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26842/perf-test/buttonNative-mount.html">643</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

File: change/@fluentui-react-8b8cd76f-59f9-49c2-8673-229b2494a10e.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "fix: move props object creation to allow for memoization",
```
Comment (khmakoto) [inline change/@fluentui-react-8b8cd76f-59f9-49c2-8673-229b2494a10e.json:3]:
```suggestion
  "comment": "fix: move props object creation to allow for memoization in DetailsList.",
```

**Review by khmakoto — APPROVED** [review]

File: change/@fluentui-react-8b8cd76f-59f9-49c2-8673-229b2494a10e.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-8b8cd76f-59f9-49c2-8673-229b2494a10e.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26842&commitId=3dd0ca15824168a0e311e2118c0734033ea17820&env=prod)

✅ There was 0 screenshots added, 2 screenshots removed, 1043 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>

Comment (spmonahan) [reply change/@fluentui-react-8b8cd76f-59f9-49c2-8673-229b2494a10e.json:1]: These tests seem to be flakey. Ignoring.

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Pull request merged**